webextension/js/popup.js

```diff
@@ -432,27 +440,46 @@ Logic.registerPanel(P_CONTAINER_EDIT, {
     });
   },
 
+  initializeRadioButtons() {
+    const colorRadioTemplate = (containerColor) => {
+      return `<input type="radio" value="${containerColor}" name="container-color" id="edit-container-panel-choose-color-${containerColor}" />
+     <label for="edit-container-panel-choose-color-${containerColor}" class="usercontext-icon choose-color-icon" data-identity-icon="circle" data-identity-color="${containerColor}">`;
+    };
+    const colors = ["blue", "turquoise", "green", "yellow", "orange", "red", "pink", "purple" ];
+    const colorRadioFieldset = document.getElementById("edit-container-panel-choose-color");
```

Comment:
Did we decide we like to use `let` when we're going to call methods on the object. (E.g., `appendChild` below)